addons/default/visiosoft/advs-module/src/Currency/CurrencyFormat.php

```diff
@@ -0,0 +1,42 @@
+<?php namespace Visiosoft\AdvsModule\Currency;
+use Anomaly\Streams\Platform\Support\Currency;
+
+class CurrencyFormat
+{
+    public function format($number, $currency = null, array $options = [])
+    {
+        $currency = strtoupper($currency ?: config('streams::currencies.default'));
+
+        $direction = array_get(
```

Comment:
Collection could be used here instead of having to many arrays ? @fatihalp

Reply:
is it edited version of money field type @mgoleli @ozcan-durak

Reply:
@fatihalp Yes. We changed with Vedat